includes/class-apple-news.php

```diff
@@ -86,4 +102,34 @@ public function migrate_settings( $settings ) {
 
 		return $migrated_settings;
 	}
+
+	/**
+	 * Displays support information for the plugin.
+	 *
+	 * @var string $format
+	 * @var boolean $with_padding
+	 * @return string
+	 * @access public
+	 */
+	public static function get_support_info( $format = 'html', $with_padding = true ) {
+		$support_info = sprintf(
+			__( 'If you need assistance with this issue, please reach out for support on <a href="%s">WordPress.org</a> or <a href="%s">github</a>.', 'apple-news' ),
+			esc_attr( self::$wordpress_org_support_url ),
+			esc_attr( self::$github_support_url )
```

Comment:
I think these 2 lines should actually be `esc_url()`, although the values are hardcoded, so not sure if any escaping is really necessary